arangodb-net-standard/Serialization/ApiClientSerialization.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.IO;
+
+namespace ArangoDBNetStandard.Serialization
+{
+    /// <summary>
+    /// The Api client serialization abastract class.
+    /// Used as a base to implement custom serialization.
+    /// </summary>
+    public abstract class ApiClientSerialization : IApiClientSerialization
+    {
+        /// <summary>
+        /// The default serialization options.
+        /// </summary>
+        protected virtual ApiClientSerializationOptions DefaultOptions => new ApiClientSerializationOptions(true, true);
```

Comment:
Do you think we can make this `public`? Consumers would then be able to set it more easily instead of having to inherit from the implementation.

Example:

```
var serializer = new JsonNetApiClientSerialization();
serializer.DefaultOptions.IgnoreNullValues = false;
```

Reply:
makes sense will do.

Reply:
Agree, this would be useful.